frontend/model/contracts/group.js

```diff
@@ -66,6 +70,17 @@ export function createInvite (
   }
 }
 
+export function getGeneralChatRoomID (chatRooms: {
+  [string]: { creator: string, name: string, editable: boolean }
+}): string | null {
+  for (const chatRoomID of Object.keys(chatRooms)) {
+    if (!chatRooms[chatRoomID].editable) {
```

Comment:
Is this really the best way to find the general chatroom? It seems hacky to me.

IMHO, it would be better to simply store the general chatroom ID directly in the group state and access it that way. Much clearer and faster, no need for a loop or checking if it's editable (which could be a property of multiple chat rooms potentially).

Reply:
Correct. I am not sure why I didn't think that way. Hahaha